config/application.rb

```diff
@@ -161,7 +161,7 @@ class Application < Rails::Application
       config.after_initialize do
         # Token used to request badges
         config.samson.badge_token = \
-          Digest::MD5.hexdigest('badge_token'.dup + Samson::Application.config.secret_key_base)
+          Digest::MD5.hexdigest('badge_token' + Samson::Application.config.secret_key_base)
```

Comment:
What was with the dups on hard-coded strings? (I'm asking for myself, not criticizing). Was this some old ruby performance thing that is no longer necessary?